src/extras/cacheAsBitmap.js

```diff
@@ -13,7 +13,7 @@ DisplayObject.prototype._cacheData = false;
  * @class
  * @ignore
  */
-class CacheData
+export default class CacheData
```

Comment:
It's referenced in the extras/index.js file, but I don't see any other references, so that reference could be removed and the import switched to a side-effect only import like the other two.